python/pyspark/sql/dataframe.py

```diff
@@ -376,24 +376,47 @@ def foreachPartition(self, f):
 
     @since(1.3)
     def cache(self):
-        """ Persists with the default storage level (C{MEMORY_ONLY}).
+        """Persists the :class:`DataFrame` with the default storage level (C{MEMORY_AND_DISK}).
```

Comment:
@rxin I updated the default in the doc, as it was actually incorrect previously.